templates/boot/generic-qemu-boot-template.jinja2

```diff
@@ -51,6 +51,11 @@ actions:
       ramdisk:
         image_arg: '-initrd {ramdisk}'
         url: {{ initrd_url }}
+{%- if dtb_url is defined and dtb_url != none %}
```

Comment:
Maybe `{%- if dtb_url %}` is enough?

Reply:
No because dtb_url is defined if equal to None